develop/reference/articles/05-theme-gulp-tasks.markdown

```diff
@@ -54,6 +61,11 @@ Here are the gulp tasks you can execute:
     Mode](/develop/tutorials/-/knowledge_base/7-0/using-developer-mode-with-themes)
     enabled.
 
-**Related Topics**
+## Liferay Theme Generator Flag options
```

Comment:
@Robert-Frampton what other flags/ configuration options are available for the theme generator? For instance, I know you can skip the update notifier and display deprecated options.

Reply:
I don't think anything else has been added.